guide/portuguese/csharp/hello-world/index.md

```diff
@@ -28,7 +29,7 @@ using System;
 ```
 
 ## Saída:
-
+```shell
 ```sh
```

Comment:
I'm wondering if this line should be removed.

Reply:
Yes, I believe ```sh should be removed.

Reply:
I have fixed this in my latest commit.